src/button.test.visuals.ts

```diff
@@ -46,41 +46,13 @@ test.describe('Button', () => {
     });
 
     test.describe('size', () => {
-      test('variant="primary"', async ({ page }, test) => {
```

Comment:
`size="large"` (the default) + `variant="primary"` is already tested by [here](https://github.com/CrowdStrike/glide-core/blob/main/src/button.test.visuals.ts#L91-L123).